notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>I think we moved this to the top namespace now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Yeah, my bad. I have used pm.sample correctly, but will remove the import.</p>